src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2152,6 +2153,17 @@ public static void copyImplementation(Design src, Design dest, boolean lockPlace
 		HashSet<String> instsWithSeparator = new HashSet<>();
 		for(Entry<String,String> e : srcToDestInstNames.entrySet()) {
 			EDIFHierCellInst cellInst = src.getNetlist().getHierCellInstFromName(e.getKey());
+			if(e.getValue().length() == 0) {
+			    // If its the top cell, remove the top cell from destNetlist
```

Comment:
Add a line to the javadoc stating that an empty destination string indicates copying to the top cell?

Reply:
Perhaps the doc should also mention that destination cell instances that are already present will be replaced?

Reply:
Updated.